poc/vdaf_poplar1.py

```diff
@@ -151,16 +151,39 @@ def shard(Poplar1, measurement, nonce, rand):
         input_shares = list(zip(keys, corr_seed, corr_inner, corr_leaf))
         return (public_share, input_shares)
 
-    def is_valid(agg_param, previous_agg_params):
+    @classmethod
+    def get_ancestor(Poplar1, input, this_level, last_level):
```

Comment:
This doesn't need to be a class method. How about defining this below the class somewhere, just as a helper function?

Reply:
Done.